### ISSUE TITLE
loadOptions doesn't work

### ISSUE THREAD

**Issue by Ali-Dalal** [collab-project/videojs-record#260]
When I try to use loadOptions, nothing seems to be changed

examples:

```
player.record().loadOptions({
  video: {
    facingMode: 'user'
  }
});
```
```
player.record().loadOptions({
  maxLength: 10
});
```

Can you help please?

**Comment by thijstriemstra** [comment]
@Ali-Dalal you need to call `getDevice` again afterwards since you're changing the camera/mic settings.

**Comment by Ali-Dalal** [comment]
@thijstriemstra I figure out the problem

I Know that I have to use ```getDevice``` in order to refresh and I was doing that.

The main problem was in the minified video-record.js file

```loadOptions``` is missing the```newOptions``` param

I manually add it to the function and add the param to merger options function

is the new option missing from the complied file?

**Comment by thijstriemstra** [comment]
@Ali-Dalal  ah I see, yes, it's not released yet and part of upcoming 2.3.2 release.
